tests-legacy/PrestaShopBundle/Controller/ControllerTest.php

```diff
@@ -109,39 +111,39 @@ public function itShouldRunTheTestedController($controllerClass)
 
     /**
      * @return array list of all legacy controllers (Back Office)
-     * If you have migrated a page on Symfony, please remove the related test.
+     *               If you have migrated a page on Symfony, please remove the related test
      */
```

Comment:
not sure why cs fixer does this 🤔

Reply:
PHP CS Fixer believes your comment is related to the line before so it aligns it 😆